pytest.ini

```diff
@@ -1,3 +1,6 @@
 [pytest]
-filterwarnings= default
-                once::DeprecationWarning
+filterwarnings = default
+                 once::DeprecationWarning
+addopts = --benchmark-skip
```

Comment:
Was this also a necessary fix to prevent segfaults?

Reply:
I don't think this was necessary to prevent segfaults, but I was super worried by seeing log messages related to the benchmark components bleeding into the other tests. Note that the benchmarking doesn't actually even run on py311, py312...

Reply:
I think I'd prefer to wait to bring back the benchmarking until we have a more structured way to track the results over time.